CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Use portable `REAL64` rather than `8` for double precision.
```

Comment:
```suggestion
```
Since this bug was never in any release, it will be confusing to have it in the log.